src/server/on-off-farm/controller.test.js

```diff
@@ -15,25 +20,27 @@ describe('#onOffFarmController', () => {
   })
 
   test('Should provide expected response', async () => {
-    const { result, statusCode } = await server.inject({
+    const { payload, statusCode } = await server.inject({
```

Comment:
this is the raw HTML not payload, so not sure payload is the best name

Reply:
We can use 'result', but result has a wider type than we'd want: 

```
export interface ServerInjectResponse<Result = object> extends ShotResponseObject {
    /**
     * the raw handler response (e.g. when not a stream or a view) before it is serialized for transmission. If not available, the value is set to payload. Useful for inspection and reuse of the
     * internal objects returned (instead of parsing the response string).
     */
    result: Result | undefined;
    /**
     * the request object.
     */
    request: Request;
}
```

Whereas payload is guaranteed to be a string: 

```
export interface ResponseObject {
    /**
     * An object containing the raw request and response objects.
     */
    raw: {
        /**
         * The simulated request object.
         */
        req: InjectedRequest;

        /**
         * The simulated response object.
         */
        res: ServerResponse;
    };

    /**
     * An object containing the response headers.
     */
    headers: OutgoingHttpHeaders;

    /**
     * The HTTP status code.
     */
    statusCode: number;

    /**
     * The HTTP status message.
     */
    statusMessage: string;

    /**
     * The payload as a UTF-8 encoded string.
     */
    payload: string;

    /**
     * The raw payload as a Buffer.
     */
    rawPayload: Buffer;

    /**
     * An object containing the response trailers
     */
    trailers: NodeJS.Dict<string>;
}
```

I don't disagree that the naming could be better, but this is the value we're after IMO.  Does that make sense to you?